WordPress/wporg-gp-translation-events
# Add is_new_contributor to attendee

Fixes #241

This PR adds a new `is_new_contributor` column to the attendees table. The value of `is_new_contributor` is calculated when the user clicks the attend button. The `is_new_contributor` field is then used in the lists of attendees that are displayed in the event details page (contributors, attendees not contributing).

Additionally, the first time a request comes to wp-admin, `is_new_contributor` will be set for all contributors of existing events.

## Testing instructions

1. Log in to wp-admin, and create a user (`user2`)
3. Create a new event that starts before now, and ends some time in the future
4. Log in as `user2`
5. Add a translation
6. Navigate to the page of the event you created in 3. and click the attend button
7. Go to the page of the event and make sure `user2` is listed as a contributor, and has the 🎉 icon
